### ISSUE TITLE
Добавление возможности сделать фотографию прямо из приложения

### ISSUE THREAD

**Issue by MariEliseeva** [vasyoid/cremebrulee#4]
Не всегда удобно загружать изображения из уже имеющихся файлов

**Comment by alyokhina-olya** [comment]
Требуется информация @MariEliseeva 
Привет! Опиши, пожалуйста, подробнее. Надо  ли предоставлять возможность пользователю сохранять сделанную фотографию локально?

**Comment by MariEliseeva** [comment]
> Требуется информация @MariEliseeva
> Привет! Опиши, пожалуйста, подробнее. Надо ли предоставлять возможность пользователю сохранять сделанную фотографию локально?
Да, сделай пожалуйста. Думаю хочется окно, в котором можно будет выбрать: хочет ли пользователь сохранять новое изображение или нет

**Comment by alyokhina-olya** [comment]
в работе

**Comment by alyokhina-olya** [comment]
решен

**Comment by alyokhina-olya** [comment]
тестируется
